quadriga/src/main/java/edu/asu/diging/quadriga/config/SecurityContext.java

```diff
@@ -35,7 +60,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .and()
                 .authorizeRequests()
                 // Anyone can access the urls
-                .antMatchers("/", "/resources/**", "/register","/login", "/loginFailed", "/register","/logout", "/reset/**").permitAll()
+                .antMatchers("/", "/resources/**", "/register","/login", "/loginFailed", "/register","/logout", "/reset/**","/rest/**").permitAll()
```

Comment:
let's not let everybody submit networks. Basic auth should be used for now until we have authentication with citesphere working.